src/dlite-misc.c

```diff
@@ -309,18 +317,37 @@ int dlite_warn(const char *msg, ...) {
 int dlite_warnx(const char *msg, ...) {
   BODY2(warnx, msg); return 0; }
 
+/* Changed below to explicitely link against _err_vformat() to avoid
+   to unintensionally link against the nonstandard BSD family of error
+   functions, on systems implementing them.
+
+   FIXME - double-check that this is the correct fix on Windows too.
+*/
+//void dlite_vfatal(int eval, const char *msg, va_list ap) {
```

Comment:
I suggest removing this code completely instead of commenting out. It leaves the source code less ambiguous for the next person that reads it.

Reply:
I have removed the commented out code lines, but left the first part of the comment as a reminder to check this on windows.